data/src/main/java/com/microsoft/azure/kusto/data/ClientRequestProperties.java

```diff
@@ -25,15 +25,15 @@
  * check out https://docs.microsoft.com/en-us/azure/kusto/api/netfx/request-properties#list-of-clientrequestproperties
  */
 public class ClientRequestProperties {
-    private static final String OPTIONS_KEY = "Options";
-    private static final String PARAMETERS_KEY = "Parameters";
     public static final String OPTION_SERVER_TIMEOUT = "servertimeout";
     public static final String OPTION_CLIENT_REQUEST_ID = "ClientRequestId";
+    public static final Pattern KUSTO_TIMESPAN_REGEX =
+            Pattern.compile("(-?)(?:(\\d+)(\\.))?(?:([0-2]?\\d)(:))?([0-5]?\\d)(:)([0-5]?\\d)(?:(\\.)(\\d+))?",
```

Comment:
I feel like a regex this big deserves a comment with an explanation and example (or a link if you took it from somewhere)

Reply:
Also I would prefer "PATTERN" over "REGEX"